variables.tf

```diff
@@ -313,7 +315,7 @@ variable "peerings" {
     ipv4_enabled                  = optional(bool, true)
     shared_key                    = optional(string, null)
     peer_asn                      = optional(number, null)
-    route_filter_id               = optional(string, null)
+    route_filter_resource_id      = optional(string, null)
```

Comment:
@jchancellor-ms Can you please update main.tf as well?

Reply:
Updated